apps/web/components/navigation/header.tsx

```diff
@@ -67,7 +67,7 @@ export function Header({ chapters, chapterSlug }: Props) {
           <DocSearch
             appId="Q9L04M9AMF"
             indexName="vibes"
-            apiKey="ef510190238e6bf2ae942046fd83e4b9"
+            apiKey="931574c95fe03313ea73bd5a77286fee"
```

Comment:
Is there a reason we're not using an environment variable for this? It feels like this should be a configuration value, even if it's `NEXT_PUBLIC_`

Reply:
Same goes for the `appId`

Reply:
Yep, we should! Updating now.